ambry-cloud/src/test/java/com.github.ambry.cloud/azure/AzureCloudDestinationTest.java

```diff
@@ -73,7 +92,7 @@ public void setup() throws Exception {
     blobId = new BlobId(BLOB_ID_V6, BlobIdType.NATIVE, dataCenterId, accountId, containerId, partitionId, false,
         BlobDataType.DATACHUNK);
 
-    azureDest = new AzureCloudDestination(mockAzureAccount);
+    azureDest = new AzureCloudDestination(mockAzureAccount, mockumentClient, "foo");
```

Comment:
`mockument`?

Reply:
Yes.  mock document = mockument.  It makes perfect sense to me!